content/docs/command-reference/exp/run.md

```diff
@@ -36,7 +36,9 @@ See the [Options](#options) section for the differences.
 </admon>
 
 Use the `--set-param` (`-S`) option as a shortcut to change
-<abbr>parameter</abbr> values [on-the-fly] before running the experiment.
+<abbr>parameter</abbr> values [on-the-fly] before running the experiment. DVC
```

Comment:
Didn't know whether to include some mention of Hydra here because this cmd Description is already huge, imo

Reply:
You can always put in subheadings. Looking at `dvc exp show`, for example, there is a lot of text before the options. I don't think the length of Description should be optimized. I'd actually favor moving more out of the `--set-params` option if possible and into Description.

Reply:
> I'd actually favor moving more out of the --set-params option if possible and into Description.

I am ok with this. 

My confusion comes from the idea that the direction was to extract content from the CMD Ref to the user guide. Expanding the description feels the opposite.

For `--set-param` (but also others like `exp show`), a similar introduction is repeated across:

- https://dvc.org/doc/start/experiments
- https://dvc.org/doc/user-guide/experiment-management/running-experiments#tuning-hyperparameters
- https://dvc.org/doc/command-reference/exp/run#description
- https://dvc.org/doc/command-reference/exp/run#-S
- https://dvc.org/doc/command-reference/exp/run#example-modify-parameters-on-the-fly

I am also ok if we agree on duplicating, was just confused about scope of each section in the docs

Reply:
Yeah, we seem to go back and forth on what belongs in the user guide vs cmd ref. I am personally fine with duplicating and not spending too much time worrying about what info belongs in each.